schema/deploy/todos.sql

```diff
@@ -0,0 +1,16 @@
+-- Deploy onboarding:todos to pg
+-- requires: appschema
+
+BEGIN;
+
+SET client_min_messages = 'warning';
+
+CREATE TABLE todo_app.todos (
+    id  SERIAL PRIMARY KEY,
```

Comment:
```suggestion
    id  GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```
This is part of the SQL standard (while `serial` is not), and ensures that you always use a generated id (unless you explicitely use `override system value` in your insert)